{"timestamp": "2025-09-15T11:35:33.332206", "level": "ERROR", "logger": "middleware", "message": "Unhandled exception occurred", "module": "middleware", "function": "handle_generic_exception", "line": 242, "exception": {"type": null, "message": null, "traceback": ["NoneType: None\n"]}, "taskName": null, "error_id": "b6b4a7aa-17ea-4b00-8b0c-19a862948d80", "error_type": "internal_error", "exception_type": "ValueError", "exception_message": "Test exception"}
{"timestamp": "2025-09-15T11:35:33.332779", "level": "ERROR", "logger": "middleware", "message": "Model error occurred", "module": "middleware", "function": "handle_model_error", "line": 302, "taskName": null, "error_id": "7c2271da-33d4-427d-b3aa-81d543b32a56", "error_type": "model_timeout", "model_name": "test-model", "exception_type": "Exception", "exception_message": "Model timeout occurred"}
{"timestamp": "2025-09-15T11:35:33.417059", "level": "ERROR", "logger": "middleware", "message": "File operation error occurred", "module": "middleware", "function": "handle_file_error", "line": 361, "taskName": null, "error_id": "7d89397f-629d-43e9-ba42-b7e98eb1e8ca", "error_type": "file_not_found", "operation": "upload", "file_id": null, "exception_type": "Exception", "exception_message": "File not found"}
{"timestamp": "2025-09-15T11:35:54.768814", "level": "ERROR", "logger": "middleware", "message": "Unhandled exception occurred", "module": "middleware", "function": "handle_generic_exception", "line": 242, "exception": {"type": null, "message": null, "traceback": ["NoneType: None\n"]}, "taskName": null, "error_id": "eeee14b7-2711-4e13-8c0e-d6702f138b54", "error_type": "internal_error", "exception_type": "ValueError", "exception_message": "Test exception"}
{"timestamp": "2025-09-15T11:35:54.769480", "level": "ERROR", "logger": "middleware", "message": "Model error occurred", "module": "middleware", "function": "handle_model_error", "line": 302, "taskName": null, "error_id": "be2d5777-5567-493a-a7f4-a781b0c2b4ed", "error_type": "model_timeout", "model_name": "test-model", "exception_type": "Exception", "exception_message": "Model timeout occurred"}
{"timestamp": "2025-09-15T11:35:54.769907", "level": "ERROR", "logger": "middleware", "message": "File operation error occurred", "module": "middleware", "function": "handle_file_error", "line": 361, "taskName": null, "error_id": "ac526a8a-b0ea-4dd0-99e5-8a01f081bcc3", "error_type": "file_not_found", "operation": "upload", "file_id": null, "exception_type": "Exception", "exception_message": "File not found"}
